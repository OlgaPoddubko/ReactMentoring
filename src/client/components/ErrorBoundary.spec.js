import React from 'react';
import ErrorBoundary from './ErrorBoundary';

describe('render ErrorBoundary', () => {
  it('should render ErrorBoundary', () => {
    const wrapper = shallow(<ErrorBoundary />);
    expect(wrapper).toMatchSnapshot();
  });
});
