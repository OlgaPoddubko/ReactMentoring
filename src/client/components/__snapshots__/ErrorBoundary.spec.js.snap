// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ErrorBoundary should render ErrorBoundary 1`] = `""`;
